src/uct/api/uct.h

```diff
@@ -1303,27 +1398,29 @@ ucs_status_t uct_iface_reject(uct_iface_h iface,
  * @ingroup UCT_RESOURCE
  * @brief Create new endpoint.
  *
- * @param [in]  iface   Interface to create the endpoint on.
+ * Create a UCT endpoint in one form available modes:
+ * -# not connected if no any type of address is present, to connect the
+ *    endpoint to remote one need to call @ref uct_ep_connect_to_ep later,
+ *    requires @ref UCT_IFACE_FLAG_CONNECT_TO_EP
+ * -# connected to remote interface if @ref uct_ep_params_t::dev_addr and
+ *    @ref uct_ep_params_t::iface_addr are set, requires iface flag
+ *    @ref UCT_IFACE_FLAG_CONNECT_TO_IFACE
+ * -# connected to socket address if @ref uct_ep_params_t::sockaddr is set,
+ *    requires iface flag @ref UCT_IFACE_FLAG_CONNECT_TO_SOCKADDR
```

Comment:
Suggested re-wording.  Please ensure I have the semantics accurately captured:

Create a UCT endpoint in one of the available modes:
-# Unconnected endpoint: If no `@ref` uct_ep_params address is present, this creates an unconnected endpoint. To establish a connection to a remote endpoint, `@ref` uct_ep_connect_to_ep will need to be called.  Use of this mode requires the `@ref` UCT_IFACE_FLAG_CONNECT_TO_EP flag to be set.
-# Connect to a remote interface:  If `@ref` uct_ep_params_t::dev_addr and `@ref` uct_ep_params_t::iface_addr are set, this will establish an endpoint that is connected to a remote interface.  This requires that the `@ref` UCT_IFACE_FLAG_CONNECT_TO_IFACE flag is set.
-# Connect to a remote socket address:  If `@ref` uct_ep_params_t::sockaddr is set, this will create an endpoint that is conected to a remote socket. This requires that the `@ref` UCT_IFACE_FLAG_CONNECT_TO_SOCKADDR flag is set.